packages/individual-packages/gen.js

```diff
@@ -1,5 +1,5 @@
 /**
- * This file is executed as part of the 'prebuild' script for this package,
```

Comment:
Move this from `packages/` to `tools/`.

Reply:
'experimental-pkg-gen' feels like a better name than 'individual-packages', no?

Reply:
Done.

Reply:
(I don't want to use the name 'experimental' here, because, like I mentioned before, we will probably include in this process [stable packages too](https://github.com/aws/aws-cdk/issues/15582))